examples/hotrod/services/route/client.go

```diff
@@ -30,19 +29,26 @@ import (
 
 // Client is a remote client that implements route.Interface
 type Client struct {
-	tracer   opentracing.Tracer
 	logger   log.Factory
 	client   *tracing.HTTPClient
 	hostPort string
 }
 
+// Transport wraps a RoundTripper. If a request is being traced with
+// Tracer, Transport will inject the current span into the headers,
+// and set HTTP related tags on the span.
+type Transport struct {
```

Comment:
This doesn't seem to add any new functionality compared to stdlib, what's the point? Previous code used custom roundtripper